app/models/concerns/form526_claim_fast_tracking_concern.rb

```diff
@@ -120,41 +117,17 @@ def diagnostic_codes
 
   def prepare_for_evss!
     begin
-      is_claim_fully_classified = update_contention_classification_all!
+      update_contention_classification_all!
```

Comment:
Removing this breaks contention-classification code, but `is_claim_fully_classified` was only used in ep merge related tasks.